team-telnyx/webrtc
# chore(WEBRTC-2021): upgrade libs to fix vulnerability issues

- upgrade libs to fix vulnerability issues

## 📝 To Do


## ✋ Manual testing

1. Run `yarn`
2. Navigate to `package/js` and run `yarn build` and `yarn link`
1. Access http://localhost:3000 from the Web dialer and link the SDK
3. Register and make a call
4. It should work as usual.

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |![image](https://user-images.githubusercontent.com/16343871/177624112-9afe9851-4a61-4195-8810-61277632913e.png)|
